src/compressor/Compressor.h

```diff
@@ -16,7 +16,10 @@
 #define CEPH_COMPRESSOR_H
 
 #include "include/int_types.h"
-#include "include/Context.h"
+#include "include/memory.h"
+#include "include/buffer.h"
```

Comment:
@tchaikov what I mean is if we do so , we should use include/buffer.h other than buffer_fwd.h in SnappyCompressor.h, there will be  both class forward and real buffer definition in SnappyCompressor.h